### PR TITLE
LPS-185744 Migrate ongoing and scheduled publications to use Dataset Display

### PR DESCRIPTION
**Pull request by pei-jung**
@xbrianlee can you add functional tests for the new ui? It can be enabled by setting `feature.flag.LPS-180155=true`.

Screen Shot 2023-06-12 at 5 35 05 PM

Screen Shot 2023-06-12 at 5 35 34 PM

/cc @drewbrokke

### CODE REVIEW

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -6785,6 +6785,11 @@
     #
     feature.flag.LPS-177194.type=release
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER8__NUMBER0__NUMBER1__NUMBER5__NUMBER5__PERIOD_TITLE
+    #
+    feature.flag.LPS-180155.title=Display Publications with Data Set Display
```
Comment (pei-jung) [inline portal-impl/src/portal.properties:6791]: Hey @david-truong, is this title ok? Also, can you provide a description if you think it's necessary to have one for this feature flag?

**Comment by xbrianlee** [comment]
Created [LPS-187075](https://issues.liferay.com/browse/LPS-187075) for the new functional test

**Comment by xbrianlee** [comment]
ci:test:publications-as

**Comment by xbrianlee** [comment]
@david-truong These changes passed testing and a test to cover this story has been added in [a53aed8](https://github.com/liferay-publications/liferay-portal/pull/602/commits/a53aed88226da997b811fe71edd254df9a09cf18). The PR can be forwarded once you finished reviewing.

**Comment by david-truong** [comment]
@pei-jung is the status and owner columns supposed to sort?

**Comment by pei-jung** [comment]
Per discussion with Dave, the status and owner columns should not be sortable.

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
@xbrianlee are the failures related?

**Comment by david-truong** [comment]
@xbrianlee this one is good from my side

**Comment by xbrianlee** [comment]
@pei-jung Can you just fix semver for `change-tracking-rest-api` ([console log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-33/test-portal-acceptance-pullrequest(master)/3842/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0))? Other than that this PR is good from my end too.

**Comment by pei-jung** [comment]
The semver has already been fixed in https://github.com/brianchandotcom/liferay-portal/pull/136214.

**Comment by pei-jung** [comment]
ci:forward

**Comment by xbrianlee** [comment]
ci:stop

**Comment by xbrianlee** [comment]
Manually forwarded
https://github.com/brianchandotcom/liferay-portal/pull/136379

**Comment by xbrianlee** [comment]
ci:stop

**Pull request closed without merging**